raiden-ts/tests/unit/reducers.spec.ts

```diff
@@ -680,8 +680,9 @@ describe('raidenReducer', () => {
   describe('transfers', () => {
     const secret = keccak256('0xdeadbeef') as Secret,
       secrethash = getSecrethash(secret),
+      fee = bigNumberify(3) as Int<32>,
```

Comment:
Reuse from fixture

Reply:
Hmm, `epicFixtures` are epic, we can't use it here because we don't have epics or depsMocks around. Will need to stay as is for now, we may refactor these constants and fixtures later.